demo/js/components/victory-zoom-container-demo.js

```diff
@@ -61,11 +61,6 @@ class CustomChart extends React.Component {
       x: [data[0].x, last(data).x]
     };
   }
-  getZoomFactor() {
```

Comment:
ended up removing this uninvoked function. not sure if this should be used in the demo and somehow got missed?